index.html

```diff
@@ -45,7 +45,7 @@ <h2 class="subtitle bold"><q>Hello! Sharing world</q></h2>
                 <p class="description">
                     A joyful celebration in the value of openess and sharing,
                     creating a positive change with people from over 10 countries
-                    is taking place in July, in Benin.
+                    is taking place in July, in Cotonou, Benin.
                 </p>
                 <div class="hero-event">
```

Comment:
### I write to you the review for this part at these lines here as I can not see all the code of the page 😊

- [ ] It will be better if you decrease the font size of the links in the mobile menu as you see it is big for the eye ,so let's decrease the size of it to make it nice for the users 🚀😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your design| 
| --- |
|  ![image](https://github.com/lily4178993/leaders-week/assets/83360636/9dfcc8f3-207d-4a6c-aec1-270b86b8cba7) |